### DISCUSSION TITLE
Saving for yearly bills

### DISCUSSION THREAD

**Discussion by Solverz-0** [firefly-iii/firefly-iii#6535] [Firefly III] [answered]
I have seen this is in the road map for #6306 but what is the current best way to put money aside each month for a yearly bill?

I can see piggy banks and auto increasing budget as options but unsure how to use them for this requirement?

Hope someone can help!

**Comment by JC5** [comment, marked as answer]
I personally save up using piggy banks. I create a piggy bank for the amount and fill that. It's not perfect but it works.

**Reply by Solverz-0** [reply]
Okay, so you create a piggy bank and then move money into it every month and when the annual bill is due you move the money from the savings account piggy bank to the expense account?

What about budgets, how would this factor into this? As normally you would set budgets for bills, would the bill budget be more for the specific month the annual bill is due or am I over complicating this?

**Reply by JC5** [reply]
Yeah that's problematic for me as well at the moment. I'm usually not budgeting for these yearly bills but by removing the necessary monthly amount from my income I get the amount I need.

**Reply by Solverz-0** [reply]
Okay thank you, I will do this for the time being then ☺ 

I mean technically, putting money aside each month in a piggy bank is sort of budgeting for the yearly bill in a sense.

I think an ideal way to combat this (feature suggestion) is to have like a hybrid budget/piggy bank or somehow link them so you can set the budget to be annual and then put money aside each month for this budget which would go into a obfuscated piggy bank? Maybe to breakdown the budget, you could have splits for budgets too for this annual use case?

Please let me know your thoughts on this ☺

**Reply by JC5** [reply]
I was thinking of something like that, yes!

**Reply by taker218** [reply]
I have the same, not with a piggy bank, but with a dedicated account (which is linked to an actual bank account).

I just add the amount of money I transfer to the bank account every month to the yearly budget. When a yearly bill (or bills that come every quarter) I deduct this amount from the yearly budget. I have a fixed amount I put away for the yearly bills. I just added up all the bills and divided this by 12. That's around the amount I transfer to the second account every month.

I have another budget for my monthly spendings (my salary minus the amount I put away for the yearly bills).

Maybe this can help you too.
